routes/user.js

```diff
@@ -60,7 +73,7 @@ router.put('/profile', (req, res) => {
 
 
 router.get('/edit-password', (req, res) => {
-   res.render('users/edit-password', { user: req.session.user });
+   res.render('users/edit-password', { user: req.session.user, userprofile: user });
```

Comment:
what is userprofile? seems to me it is the same as the req.session.user? or just use the user variable from the database without the user, they refer to the same entity